### DISCUSSION TITLE
AIS Catcher remote site not reconnecting

### DISCUSSION THREAD

**Discussion by Gazman429** [jvde-github/AIS-catcher#255] [Q&A]
Asking for suggestions, I have a number of remote ADS-B receiver sites that I plan on adding AIS receivers to some of these sites, one of my local soak testing sites where I have an AIS setup in I'm having problems with AIS Catcher not reconnecting to my server after either an internet interruption / outage or simulated outage as part of my equipment soak testing, now my ADS-B reconnects without fail but AIS Catcher will not reconnect after a period of outage time, I've also tried AIS Dispatcher as an in between but it too errors and will not reconnect.
Is there a setting in AIS Catcher that would solve the issue or other low impact 3rd party software that someone might know or try that will solve or might solve the reconnection issue, I can remote in and restart but I really don't want to have to do that every time, any suggestions greatly appreciated.
Gary.

**Comment by jvde-github** [comment]
Hi Gary, Are you sending by UDP or TCP to the server?

**Reply by Gazman429** [reply]
UDP

**Comment by jvde-github** [comment]
The simplest trick might be to use `-u xxxx port RESET 5`. This will actually recreate the socket on the sender every 5 minutes. Not sure why this would solve it but it has shown to solve similar type of problems for other users where the internet provider resets a connection.
In a way it is a flying reset of the UDP connection without resetting the whole program (which can also be done with -T). 

Let me know if this helps?

**Reply by Gazman429** [reply]
Ok thanks' I shall give that a try and let you know how it goes.

**Reply by jvde-github** [reply]
Hopefully the connection is no longer down than 5 minutes this way

**Reply by Gazman429** [reply]
OK, I setup 2 side by side feeding back to my server, one without the RESET and the other with, simulated 2 quick test outages, first was 6 minutes both reconnected on the 6 minutes, second test of 22 minutes the one without failed to reconnect but the one with RESET reconnected straight away, going to do longer outage tests of various types but I think that has solved the reconnection issue.
I will let you know on follow up testing.
Thanks so much for your help Jasper.
Gary.
